src/raftstore/store/fsm/apply.rs

```diff
@@ -1805,26 +1799,17 @@ impl ApplyDelegate {
             );
 
             // Sends message to the source apply worker and pause `exec_commit_merge` process
-            if let Some(mailbox) = ctx.router.mailbox(self.region_id()) {
```

Comment:
Is it possible that when the task is sent back, the mailbox is changed? For example, target peer is removed and readded.

Reply:
Even that, seems it's okay to send `LogsUpToDate` to newly readded target peer, cause `LogsUpToDate` is just an empty message which is only used to trigger target peer to check `resume_pending_merge`, so it won't make any effect on the target peer.